lib/debug/src/frame.rs

```diff
@@ -0,0 +1,322 @@
+#![allow(non_upper_case_globals)]
+
+use cranelift_codegen::isa::{CallConv, RegUnit, TargetIsa};
+use cranelift_entity::EntityRef;
+use cranelift_wasm::DefinedFuncIndex;
+use std::collections::HashMap;
+use std::vec::Vec;
+use wasmtime_environ::{FrameLayoutCommand, FrameLayouts};
+
+use gimli::write::{EndianVec, Error};
+use gimli::{LittleEndian, Register, X86_64};
+
+trait Writer: gimli::write::Writer<Endian = LittleEndian> {}
+impl Writer for EndianVec<LittleEndian> {}
+
+enum CallFrameInstruction {
```

Comment:
Could you open a gimli PR for this and the rest of the non cranelift specific parts of this file, so that I could use it in https://github.com/bjorn3/rustc_codegen_cranelift?